app/src/autoupdate.js

```diff
@@ -243,35 +243,36 @@ const onUpdateAvailable = (info) => {
   // Prompt that a new version is available when using an installed app, or the release page is configured.
   const targetWindow = getTargetWindow();
   if (targetWindow && (!process.env.PORTABLE_EXECUTABLE_DIR || config.has('electron.releaseUrl'))) {
-    const index = dialog.showMessageBoxSync(targetWindow, {
+    dialog.showMessageBox(targetWindow, {
```

Comment:
From a cursory look, it seems fine since this call to onUpdateAvailable is event-based, but since this is no longer a blocking (synchronous) message box, is there anything that happens along with other handlers or down-the-line with the 'update-available' event that had needed to be blocked?

Reply:
The only thing that required it to be synchronous was the check on which button was pressed, which is now inside the `Promise.then` block.

Reply:
I think any weirdness can be resolved by using `await` but the linter was flipping out about me doing `value = await blah blah` despite electron docs indicating that was valid. I didn't want to push it.